tests/test_unhandled_record.py

```diff
@@ -0,0 +1,16 @@
+from shipwright.cli import switch
+
+
+example = {
```

Comment:
I got a record that looks like this, it wasn't handled.

I'd rather just see the JSON dumps of this, but if you think it should print something else, that would be cool too.